exersice1.js

```diff
@@ -0,0 +1,100 @@
+// You are talking to a friend who suddently looks at you 
+// and asks: "How old will you be in 2045?" Hmm you remember 
+// the year you were born and try to do some calculation:
+// Born in 1987 + 3 that's 1990. 90 - 40 is 50 + 5 thats 58. 
+// I will be 58! Wow that was painful! Let's fix that by making 
+// some code that automatically does this!
+
+let yearOfBirth = 1986;
+let yearFuture = 2051;
+let age;
+
+age = yearFuture - yearOfBirth;
+
+console.log(`You will be ${age} old in ${yearFuture}`);
+
+// Goodboy-Oldboy (A dog age calculator)
+// The same friend(who by the way loves dogs) asks how old his 
+// dog will be in 2045. Hmm you think, lets make this into a
+// product as well! Dogs age can either be measured in dog years
+// or in human years, this we want to take into consideration!
+
+const dogYearOfBirth = 2022;
+const dogYearFuture = 2025;
+let dogYear;
+
+dogYear = dogYearFuture - dogYearOfBirth;
+
+const shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears === false) {
+  console.log(`Your dog will be ${dogYear * 7} dog years old in ${dogYear}`);
+} else {
+  console.log(`Your dog will be ${dogYear} human years old in ${dogYear}`);
+}
+
+
+// Housey pricey (A house price estimator)
+// Two of your friends are considering buying a house, but cannot 
+// figure out what the right price should be. Your friends know the
+// width, the height and the depth of the house and the garden
+// size.Lets help them figure out if they paid too much:
+
+const widthHouse = 8;
+const heightHouse = 10;
+const depthHouse = 10;
+const gardenSize = 100;
+
+//housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+const peterHouseParameters = {width: 8, heigth: 10, deep: 10, garden: 100, price: 2500000}
+let housePricePeter = peterHouseParameters.width * peterHouseParameters.heigth* peterHouseParameters.deep * 2.5 * 1000 + peterHouseParameters.garden * 300;
+console.log("house Price Peter", housePricePeter);
+if (housePricePeter > peterHouseParameters.price) {
```

Comment:
I love the approach you have chosen! 👏 However, I believe the condition should be reversed, correct? If the real price (housePricePeter) is lower, then that should indicate that the price is too high.